### PR TITLE
adding diagram to readme file

### PR DESCRIPTION
**Pull request by Asafh7**
adding diagram to readme file

### CODE REVIEW

**Review by avraham1992 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,3 +33,7 @@ Falcon is designed to help you manage your team's tasks and controls workflows i
 # The Team
 
 ![OurTeam](./static/img/OurTeam.jpeg)
+
+# Workflow Diagram
```
Comment (AvivLiberman) [inline README.md:37]: Should be called DB diagram, it is not exactly workflow diagram

**Review by AvivLiberman — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (orzionpour) [reply README.md:37]: At the SOW they are calling it ERD
https://github.com/redhat-beyond/documents/blob/main/Beyond06-SOW.md

**Review by orzionpour — APPROVED** [review]

**Review by AvivLiberman — APPROVED** [review]

**Review by orzionpour — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -33,3 +33,7 @@ Falcon is designed to help you manage your team's tasks and controls workflows i
 # The Team
 
 ![OurTeam](./static/img/OurTeam.jpeg)
+
+# ER Diagram
```
Comment (natifridman) [inline README.md:37]: I think that the most common term is ERD

**Review by natifridman — CHANGES_REQUESTED** [review]

**Comment by nunnatsa** [comment]
@Asafh7 - can you bring the diagram into the project as a picture, and embed it in the document, so the reader would able to see it without clicking the link?

image


image

**Comment by orzionpour** [comment]
Better to squash the commits

**Comment by orzionpour** [comment]
Better squash the commits

**Comment by nunnatsa** [comment]
@Asafh7, you'll need to add the image fule to git and use the right format to embed it. See an example just a few lines above, in the team picture.

**Comment by mgold1234** [comment]
@Asafh7, you need to put the image in git/falcon/static/img folder like all other pictures in this page. and as @nunnatsa said it should be in the right format.

**Comment by AvivLiberman** [comment]
I think you should leave the link because when we will change some tables in the future you will need to change the picture here as well

**Comment by mgold1234** [comment]
@AvivLiberman not a link, he need to add the path for the picture in img folder

**Comment by nunnatsa** [comment]
@AvivLiberman, I get your point, but I still think it's better to have an embedded image than a link. And it's very easy to create this image too. 

Another (a bit stupid, I admit) reason is that you change the diagram before you push the relevant code, so for a short period, the diagram does not match the actual code in main.

**Review by nunnatsa — APPROVED** [review]

**Pull request merged**